web/src/common/index.ts

```diff
@@ -34,3 +34,19 @@ export function roadLineWidth(extraWidth: number): ExpressionSpecification {
     24 + extraWidth,
   ];
 }
+
+export function prettyPrintDistance(meters: number): string {
+  if (meters < 1000.0) {
```

Comment:
I admit I'm completely ignorant, but I see all the speed limits are in miles. So should we be printing miles here?

Reply:
https://www.reddit.com/r/AskABrit/comments/lcx4do/do_brits_use_miles/
... I went down a rabbit hole here. I guess I've just gotten a bit used to mixing systems here. Speeds are definitely mph, but I'm a bit unclear whether it's more conventional to use m/km or miles. Will check, thanks for flagging